daemon/src/main/java/org/sonarlint/daemon/interceptors/ExceptionInterceptor.java

```diff
@@ -19,24 +19,24 @@
  */
 package org.sonarlint.daemon.interceptors;
 
+import io.grpc.ForwardingServerCall.SimpleForwardingServerCall;
 import io.grpc.Metadata;
-import io.grpc.MethodDescriptor;
 import io.grpc.ServerCall;
 import io.grpc.ServerCall.Listener;
 import io.grpc.ServerCallHandler;
 import io.grpc.ServerInterceptor;
-import io.grpc.ForwardingServerCall.SimpleForwardingServerCall;
 import io.grpc.Status;
 
 public class ExceptionInterceptor implements ServerInterceptor {
+
   @Override
-  public <ReqT, RespT> Listener<ReqT> interceptCall(MethodDescriptor<ReqT, RespT> method, ServerCall<RespT> call, Metadata headers, ServerCallHandler<ReqT, RespT> next) {
-    ServerCall<RespT> serverCall = new TransformStatusServerCall<>(call);
-    return next.startCall(method, serverCall, headers);
+  public <ReqT, RespT> Listener<ReqT> interceptCall(ServerCall<ReqT, RespT> call, Metadata headers, ServerCallHandler<ReqT, RespT> next) {
+    ServerCall<ReqT, RespT> serverCall = new TransformStatusServerCall<>(call);
+    return next.startCall(serverCall, headers);
   }
 
-  static class TransformStatusServerCall<RespT> extends SimpleForwardingServerCall<RespT> {
-    protected TransformStatusServerCall(ServerCall<RespT> delegate) {
+  static class TransformStatusServerCall<ReqT, RespT> extends SimpleForwardingServerCall<ReqT, RespT> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS00119)